source/index.html

```diff
@@ -129,12 +129,16 @@ <h1 class="row-title" id="dinner">Top Dinner Recipes</h1>
         </div>
       </section>
 
-      <section class="search-results hidden">
+      <section class="search-results hidden" id="search-results-page">
         <!-- TODO: Populate query field -->
-        <h1>Top recipes for "bread"</h1>
-        <div class="results-grid">
+        <h1 id="search-results-title">Top recipes for "bread"</h1>
+        <div class="results-grid" id="results-grid-recipe-card-location">
```

Comment:
this id is really long can we make it simpler just like "search-results-container" or something

Reply:
Renamed 'results-grid-recipe-card-location' to 'search-results-container', good naming